specs/interop/supervisor.md

```diff
@@ -74,6 +75,8 @@ Specifically, this helps apply message-expiry rules on message checks.
 
 Object:
 - `timestamp`: `HexUint64`
+- `timeout`: `HexUint64` or `null` - optional, requests verification to still hold at `timestamp+timeout`
+  (message expiry may drop previously valid messages).
```

Comment:
would love an explanation on this change

Reply:
Added additional explanation